### PR TITLE
readme: Update README.rst to align with 1.1.0 release

### PR DESCRIPTION
**Pull request by jeez**
Signed-off-by: Malcolm Prinn malcolm.prinn@intel.com
Signed-off-by: Jesus Sanchez-Palencia jesus.sanchez-palencia@intel.com

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -176,46 +196,179 @@ For flashing the board OpenOCD must be used. You can optionally use gdb
...
 or
 
-``$ screen /dev/ttyUSB0 115200``
+``$ screen /dev/ttyUSBXXX 115200``
+
+Where /dev/ttyUSBXXX is the path to the attached UART device.
+e.g. /dev/ttyUSB0
 
 
 Known Issues and Workarounds
```
Comment (mprinn) [inline README.rst:259]: To quote you on an earlier commit Jesus

"Let's make it clear to which version we refer to here. What about something like: "Known Issues and Workarounds - QMSI Release 1.1.0"? Or even "Known Issues and Workarounds" "Affected version: QMSI 1.1.0" ?"

;)

Comment (jeez) [inline README.rst:259]: fixing now, thanks

Comment (jeez) [inline README.rst:259]: fixed!

**Comment by dalessan** [comment]
LGTM

**Comment by mcomyns** [comment]
Looks fine for me...

**Pull request merged**